metricbeat/module/kubernetes/_meta/test/docs/01_playground/metricbeat.yaml

```diff
@@ -27,19 +27,7 @@ data:
                   period: 10s
                   add_metadata: true
                   metricsets:
-                    - state_node
-                    - state_deployment
-                    - state_daemonset
-                    - state_replicaset
-                    - state_pod
-                    - state_container
-                    - state_cronjob
-                    - state_resourcequota
-                    - state_statefulset
-                    - state_service
-                    - state_persistentvolume
-                    - state_persistentvolumeclaim
-                    - state_storageclass
+                    - state_namespace
```

Comment:
seems you pushed local changes for this file, could you please revert?

Reply:
Yes. Is it ok now with just `state_namespace` added?